utils/js/Decorators.js

```diff
@@ -46,16 +41,11 @@ export function computeOnce(target, key, descriptor) {
 
     const isMethod = isFunction(value),
         baseFnName = isMethod ? 'value' : 'get',
-        baseFn = isMethod ? value : get,
-        cacheKey = '_xh_' + key;
+        baseFn = isMethod ? value : get;
     return {
         ...descriptor,
         [baseFnName]: function() {
-            let val = this[cacheKey];
-            if (isUndefined(val)) {
-                val = this[cacheKey] = baseFn.call(this);
-            }
-            return val;
+            return getOrCreate(this, '_xh_' + key, () => baseFn.call(this));
```

Comment:
Noting for the record here that I haven't yet taken the time to really look at / grok these changes. Making this comment for myself more than anything, as they look like interesting uses of this new `getOrCreate` util!